src/display.rs

```diff
@@ -142,15 +142,20 @@ impl Display {
         // get window properties for initializing the other subsystems
         let mut viewport_size = window.inner_size_pixels()
             .expect("glutin returns window size");
-        let dpr = window.hidpi_factor();
+        let dpr = if config.font().scale_with_dpi() {
+            window.hidpi_factor()
+        } else {
+            1.0
+        };
 
         info!("device_pixel_ratio: {}", dpr);
+        println!("DPR: {}", dpr);
```

Comment:
:bomb:

Reply:
Ooopsie, I knew I was gonna forget removing this, thanks!